projects/demo/src/environments/environment.ts

```diff
@@ -4,6 +4,23 @@ import { EnvironmentOptions } from '@igo2/integration';
 export const environment: EnvironmentOptions = {
   production: false,
   igo: {
+    directionsSources: [
+      {
+        osrm: {
+          name: 'OSRM Québec (Public)',
+          type: 'public',
+          url: '/apis/itineraire/route/v1/driving/'
+        }
+      },
+      {
+        osrm: {
+          name: 'OSRM Québec (Partenaires)',
+          type: 'private',
+          url: '/apis/itineraire/route/v1/forestier/',
+          authorizationUrl: '/apis/igo2/user/igo'
+        }
+      }
```

Comment:
Après discussion avec le MTQ, est-ce que ce serait une bonne idée d'avoir une seule config d'OSRM, avec différents profils par exemple:
```suggestion
      {
        osrm: {
          name: 'OSRM Québec',
          type: 'public',
          url: '/apis/itineraire/route/v1', // URL concaténé avec le PROFIL_NAME
          profil: [
            { name: 'driving' },
            {
              name: 'forestier',
              authorization: {
                url: '/apis/igo2/user/igo',
                property: 'hasOsrmPrivateAccess'
              }
            }
          ]
        }
```